plugins/common/common.cpp

```diff
@@ -1298,11 +1312,15 @@ void AuiToolBar::OnTool( wxCommandEvent& event )
 		return;
 	}
 
-	wxObject* wxobject = (wxObject*) tb->FindTool( event.GetId() )->GetUserData();
-
-	if ( NULL != wxobject )
+	wxAuiToolBarItem* item = tb->FindTool(event.GetId());
+	if (item)
 	{
-		m_manager->SelectObject( wxobject );
+		wxObject* wxobject = (wxObject*) GetPtrData(item->GetUserData());
+
+		if (NULL != wxobject)
```

Comment:
Modern-C++: Don't use `NULL` for pointer values and don't compare pointers explicit to `nullptr`, a simple implicit test `if (wxobject)` is preferred